src/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  background-color: black;
+  margin: 0;
+}
+
+.grey-box {
+  padding: 10px;
+  margin: 10px;
+  background-color: #dcdcdc;
+}
+
+.yellow-box {
+  margin: 10px;
+  background-color: #ffa000;
+}
+
+.blue-box {
+  box-sizing: border-box;
```

Comment:
add box-sizing: border-box rule to all elements on the page (use * selector for this)